articles/153_roslaunch_remote_launch.md

```diff
@@ -0,0 +1,185 @@
+# Multi-Machine and Remote Launching
+## Goals
+- Connect to a remote host and run nodes on it
+    - Ensure this capability does not compromise security
+- Support arbitrary remote execution methods
+- Monitor the status of and manage lifecycles of nodes across systems
+- Shutdown nodes remotely
+- Ability to detach the machine used to launch a system and have the system continue running
+    - Be able to reconnect to the system to monitor and manage nodes running remotely
+
+
+To achieve our goals, this update will consist of two primary components: an ExecuteRemote action, and a LaunchServiceNode.
+
+The ExecuteRemote action will build off of the refactor he is working on described here: https://github.com/ros2/design/pull/272
+Since we don't want to limit remote execution to a single protocol, ExecuteRemote will be a base class and we will create an implementation using SSH which we will call ExecuteRemoteSSH.
+Remote machines will be described by a Machine class which will also be a base class for protocol-specific implementations.
+Together these components will provide the means to execute processes remotely and should cover most basic use cases.
+
+The LaunchServiceNode(s) will expose ROS2 services and topics for handling startup and shutdown of processes on remote machines.
```

Comment:
This LaunchServiceNode could be designed as an independent component - that is possible to use purely locally to detach a launch command from an ongoing launch context.